ocaml/utils/compilation_unit.ml

```diff
@@ -170,74 +173,233 @@ end = struct
 end
 
 module T0 : sig
-  type t
+  type descr = private
+    { name : Name.t;
+      for_pack_prefix : Prefix.t;
+      arguments : (t * t) list
```

Comment:
There are comments saying "By "compilation unit" we mean the code and data associated with the
   compilation of a single .ml source file".

Is that true? I think a single `.ml` soulce file cannot have arguments. It's only by `-instantiate` command where you can instantiate a parameterized `.ml` source file?

Reply:
There still is a compilation unit that describes just the .ml file without any arguments (it has the instantiating functor in it). Instances are a funny special case, though, so I added a bit to the comment in question.